mscore/synthcontrol.cpp

```diff
@@ -391,6 +395,7 @@ void SynthControl::recallButtonClicked()
             else
                   e.unknown();
             }
+      state.setIsDefault(false);
```

Comment:
Isn't that button supposed to revert all to default settings? Or what does it do really?

Reply:
yes, you're right. This line shouldn't be here, I don't think.

Reply:
Sorry to revive this, but shouldn't that line then be `state.setIsDefault(true)`? The state is being read via `SynthesizerState::read()` function which would mark it non-default too.

Reply:
Yeah, probably. You know what, that's probably what I meant in the first place by putting that line there, it should have had true instead of false.

Reply:
Fixed.

Reply:
Travis has failed, apparently a typo is there: https://travis-ci.org/musescore/MuseScore/jobs/545635834#L2576

Reply:
oops, I didn't take the time to build it. I used -> instead of .

Reply:
Should be fixed now.